index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="container">
+      <p>Alex Hamn</p>
```

Comment:
Following the [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp).
 `Your name` should be wrap in an `h1` tag which is the main title of the page so it is good practice to use `h1` tag.